features/prompt-registry/snippets.mdx

```diff
@@ -48,6 +48,11 @@ When you run a parent template that contains snippets, the system 'renders' the
 
 *Rendering snippets from the Playground*
 
+### Webhooks
+
+When you update a template which is being referenced on another template by a snippet, both of them get a webhook update. 
```

Comment:
When you update a template that's included as a snippet in another template, both webhook events will be triggered for both templates.